quattor/types/aquilon/system.pan

```diff
@@ -55,9 +55,11 @@ type structure_archetype_os = {
 };
 
 type structure_archetype = {
-    "name"          : string # e.g. "aquilon"
+    "name"          ? string # e.g. "aquilon"
     @{ Details of operating system as defined by aquilon broker }
-    "os"            ? structure_archetype_os
+    "os"            : structure_archetype_os
```

Comment:
Please revert this to `"os"            : string # e.g. "linux"` since the broker has been generating this prior to the the change that introduced this dict.

Reply:
See the complementary PR https://github.com/quattor/aquilon/pull/94 for the broker...